README.md

```diff
@@ -1,12 +1,12 @@
-## OpenSearch AI Flow Framework
+## OpenSearch Flow Framework
 
-This project is an OpenSearch plugin that enables builders to innovate AI apps on OpenSearch.
+This project is an OpenSearch plugin that enables builders to innovate AI applications on OpenSearch.
 
 The current process of using ML offerings in OpenSearch, such as Semantic Search, requires users to handle complex setup and pre-processing tasks, and send verbose user queries, both of which can be time-consuming and error-prone.
 
-The directional idea is to provide OpenSearch users with use case templates, which provide a compact description (e.g., JSON document). These templates would describe configurations for automated workflows such as Retrieval Augment Generation (RAG), AI connectors and other components that prime OpenSearch as a backend to leverage generative models—once primed, builders can query OpenSearch directly without building middleware logic to stitch together data flows and ML models.
+We want to introduce our customers to a new no-code/low-code builder experience ([Backend RFC](https://github.com/opensearch-project/OpenSearch/issues/9213) and [Frontend RFC](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/4755)) that empowers users to compose AI-augmented query and ingestion flows, integrate ML models supported by ML-Commons, and streamline the OpenSearch app development experience through a drag-and-drop designer.  The front end will help users create use case templates, which provide a compact description of configuration steps for automated workflows such as Retrieval Augment Generation (RAG), AI connectors and other components that prime OpenSearch as a backend to leverage generative models.  Once primed, builders can query OpenSearch directly without building middleware logic to stitch together data flows and ML models.
```

Comment:
Should we add a statement that this is more then just ML and can power many different workflows in opensearch, relating to alerts, other plugins, observability and etc to make it clear we don't want to only onboard AI/ML use cases.